packages/common/src/doctype.ts

```diff
@@ -18,56 +18,85 @@ export enum AnchorStatus {
     NOT_REQUESTED, PENDING, PROCESSING, ANCHORED, FAILED
 }
 
-/**
- * Describes anchor record
- */
-export interface AnchorRecord {
-    prev: CID; // should be CID type
-    proof: CID; // should be CID type
-    path: string;
+export interface RecordHeader {
+    controllers: Array<string>
+    schema?: string
+    tags?: Array<string>
+
+    unique?: string
+
+    [index: string]: any // allow support for future changes
+}
+
+export type GenesisHeader = RecordHeader
+
+export type GenesisRecord = {
+    header: GenesisHeader
+    data?: any
+    unique?: string
+}
+
+export interface UnsignedRecord {
+    id: CID
+    header?: RecordHeader
+    data: any
+    prev: CID
 }
 
-/**
- * Describes anchor proof
- */
 export interface AnchorProof {
-    chainId: string;
-    blockNumber: number;
-    blockTimestamp: number;
-    txHash: CID;
-    root: CID;
+    chainId: string,
+    blockNumber: number,
+    blockTimestamp: number,
+    txHash: CID,
+    root: CID,
+}
+
+export interface AnchorRecord {
+    id: CID,
+    prev: CID,
+    refs?: Array<CID>
```

Comment:
This refs field isn't actually used anywhere yet, right?  Can we remove it for now until we actually implement logarithmic document syncing?